software/util/reorg.py

```diff
@@ -6,67 +6,104 @@
 """
 
 from collections.abc import Sequence
-from rdflib import Graph, Namespace
+from rdflib import Graph, Namespace, RDF, RDFS, DCTERMS, URIRef
```

Comment:
See the Python style guide, only import packages, not classes.

Reply:
Done